sql/scriptdev2/scriptdev2.sql

```diff
@@ -621,6 +621,8 @@ UPDATE creature_template SET ScriptName='boss_tethyr' WHERE entry=23899;
 UPDATE creature_template SET ScriptName='npc_major_mills' WHERE entry=23905;
 UPDATE creature_template SET ScriptName='mob_invis_firework_helper' WHERE entry=24025;
 UPDATE creature_template SET ScriptName='npc_smolderwing' WHERE entry=23789;
+UPDATE creature_template SET ScriptName='npc_theramore_practicing_guard' WHERE entry=4951;
+UPDATE creature_template SET HealthMultiplier='1', ScriptName='npc_theramore_combat_dummy' WHERE entry=4952;
```

Comment:
This HP multi should be in DB

Reply:
The HP Multi in DB is the one that was sniffed to make the NPCs HP be `1` (so in DB it was `0.021` or something similar to that). On the reference client, the dummy has 42 HP, which corresponds to a multiplier of `1`. The problem with giving the dummy the `0.021` multiplier as it is currently is that any method of death-prevention would cause the server to send a damage value of `0` to clients, which will be interpreted as a miss, dodge, resist, etc. so it doesn't do the correct "getting-attacked animation" (well the visual animation is correct, but because it's all misses and dodges, it will always just "whoosh" instead of the response attacks get when they actually connect.

Reply:
If you want I can also edit this multiplier value in the db repo and create a separate PR over there, but it currently conflicts with sniff-data